config/locales/en/providers.yml

```diff
@@ -478,6 +479,15 @@ en:
         uploaded: Uploaded
         title: Files added
         no_files: Files uploaded will appear here.
+    no_eligibility_assessments:
+      show:
+        page_title: A caseworker needs to review your application
+        no_cfe_assessment: A caseworker needs to review your application
+        heading: A caseworker needs to review your application
+        cannot_calculate: We cannot calculate your client's financial eligibility yet because you uploaded bank statements
```

Comment:
this is the most minor nitpick ever but should there be a full stop here?

Reply:
will make this change on the next ticket